src/components/__tests__/ThoughtDelete.js

```diff
@@ -0,0 +1,23 @@
+import { act } from 'react-dom/test-utils'
+
+it('delete thought', async () => {
```

Comment:
This looks good, and we will need a test for the `delete` shortcut which deletes a non-empty thought.